dydx3/modules/private.py

```diff
@@ -822,6 +823,92 @@ def create_withdrawal(
         }
         return self._post('withdrawals', params)
 
+    def create_transfer(
+        self,
+        amount,
+        sender_position_id,
+        receiver_account_id,
+        receiver_public_key,
+        receiver_position_id,
+        client_id=None,
+        expiration=None,
+        expiration_epoch_seconds=None,
+        signature=None,
+    ):
+        '''
+        Create a L2 transfer.
+
+        :param amount: required
+        :type amount: str
+
+        :param sender_position_id: required
+        :type sender_position_id: int or str
+
+        :param receiver_account_id: required
+        :type receiver_account_id: str
+
+        :param receiver_public_key: required
+        :type receiver_public_key: str
+
+        :param receiver_position_id: required
+        :type receiver_position_id: int or str
+
+        :param client_id: optional
+        :type client_id: str
+
+        :param expiration: optional
+        :type expiration: ISO str
+
+        :param expiration_epoch_seconds: optional
+        :type expiration_epoch_seconds: int
+
+        :param signature: optional
+        :type signature: str
+
+        :returns: Transfer
+
+        :raises: DydxAPIError
+        '''
+        client_id = client_id or random_client_id()
+
+        if bool(expiration) == bool(expiration_epoch_seconds):
+            raise ValueError(
+                'Exactly one of expiration and expiration_epoch_seconds must '
+                'be specified',
+            )
+        expiration = expiration or epoch_seconds_to_iso(
+            expiration_epoch_seconds,
+        )
+        expiration_epoch_seconds = (
+            expiration_epoch_seconds or iso_to_epoch_seconds(expiration)
+        )
+
```

Comment:
we should allow passing in a signature like we do for create_order

Reply:
It's not a good pattern to mutate args

Reply:
Done.